addon-test-support/@ember/test-helpers/dom/double-click.ts

```diff
@@ -80,6 +82,9 @@ export function __doubleClick__(element: Element | Document, options: MouseEvent
   doubleClick('button', { shiftKey: true });
 */
 export default function doubleClick(target: Target, options: MouseEventInit = {}): Promise<void> {
+
+  const mergedOptions = assign({ buttons: MOUSE_EVENT_BUTTONS.PRIMARY }, options);
```

Comment:
Can we make a constant above for `DEFAULT_CLICK_OPTIONS` and tweak this to:

```suggestion
  const options = assign({}, DEFAULT_CLICK_OPTIONS, _options);
```